### PR TITLE
add sources of 3rd-party extensions to the FAQ

### LINKED ISSUE #2103: Fishshell Frameworks
What do you think about adding links (perhaps in the Wiki) to some of the known Fishshell _frameworks_ out there, such as [Fisherman](http://fisherman.sh)?

### PR DESCRIPTION
**Pull request by krader1961**
Fixes #2103

### CODE REVIEW

**Comment by faho** [comment]
I'm okay with this, but not particularly in love.

What I like is that you picked alphabetical ordering - that's how it should be, because we don't want to pick favorites.

**Comment by krader1961** [comment]
Alphabetical was the only safe choice since we really shouldn't be endorsing one of the available options over the others. As for the verbiage it won't hurt my feelings at all if someone wants to propose something different. What I wrote was just to get words on paper to force people to think about what we want to say on the topic.

File: doc_src/faq.hdr [hunk 1]
```diff
@@ -220,6 +221,11 @@ rm -f fish fish_indent
 
 Fish reserves the <a href="http://www.unicode.org/faq/private_use.html">Unicode private-use character range</a> from U+F600 thru U+F73F for internal use. Any attempt to feed characters in that range to fish will result in them being replaced by the Unicode "replacement character" U+FFFD. This includes both interactive input as well as any file read by fish (but not programs run by fish).
 
+<hr>
+\section faq-third-party Where can I find 3rd-party extensions?
```
Comment (zanchey) [inline doc_src/faq.hdr]: Although we know them as third-party extensions, I don't think this question means much to new users. What about "Where can I find extra tools for fish?"

File: doc_src/faq.hdr [hunk 2]
```diff
@@ -220,6 +221,11 @@ rm -f fish fish_indent
...
+
+The fish user community extends fish in unique and useful ways that aren't always appropriate for inclusion in the core fish distribution. You can find those extensions in various third-party repositories and fish package managers. These include <a href="https://github.com/fisherman/fisherman">Fisherman</a>, <a href="https://github.com/tuvistavie/fundle">Fundle</a>, <a href="https://github.com/oh-my-fish/oh-my-fish">Oh My Fish</a>, <a href="https://github.com/justinmayer/tacklebox">Tacklebox</a>. This is not an exhaustive list and the fish project has no opinion regarding the merits of the various options.
```
Comment (zanchey) [inline doc_src/faq.hdr]: how about "...find those extensions, including prompts, themes and useful functions, in various third-party repositories"?

**Comment by krader1961** [comment]
Thanks for the suggestions. I've incorporated all of it and added some additional verbiage of my own. PTAL.

**Comment by zanchey** [comment]
Looks fine to me!

**Comment by krader1961** [comment]
Closed with merge of b12c413c281e3569c9f63bc6a7ffeab0a11e868f.

**Pull request closed without merging**